### PR TITLE
Publish instructions for keypresses in the keyboard_controller.

### PR DESCRIPTION
**Pull request by tigressine**

### CODE REVIEW

File: packages/communication/ezrassor_keyboard_controller/launch/keyboard_controller.launch [hunk 1]
```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Launch the keyboard controller. Written by Tiger Sachse. -->
 <launch>
   <arg name="wheel_instructions_topic"/>
   <arg name="front_arm_instructions_topic"/>
-  <arg name="back_arm_instructions_topic"/>
+  <arg name="rear_arm_instructions_topic"/>
```
Comment (CSharpRon) [inline packages/communication/ezrassor_keyboard_controller/launch/keyboard_controller.launch]: I think you should go back to using "back". The ezrassor_sim_control refers to the arms and drums by front and back.

File: packages/communication/ezrassor_keyboard_controller/nodes/keyboard_controller [hunk 2]
```diff
@@ -1,8 +1,195 @@
...
+)
+WHEELS_STOP_INSTRUCTION = geometry_msgs.msg.Twist(
+    linear=geometry_msgs.msg.Vector3(x=0.0, y=0.0, z=0.0),
+    angular=geometry_msgs.msg.Vector3(x=0.0, y=0.0, z=0.0),
+)
+
+FRONT_ARM_UP_INSTRUCTION = std_msgs.msg.Float32(1.0)
+FRONT_ARM_DOWN_INSTRUCTION = std_msgs.msg.Float32(-1.0)
+FRONT_ARM_STOP_INSTRUCTION = std_msgs.msg.Float32(0.0)
+REAR_ARM_UP_INSTRUCTION = std_msgs.msg.Float32(1.0)
```
Comment (CSharpRon) [inline packages/communication/ezrassor_keyboard_controller/nodes/keyboard_controller]: Again  **back** would probably be a good choice for consistency [see ezrassor_sim_control/config]

**Review by CSharpRon — CHANGES_REQUESTED** [review]
Clean changes! There was only one change I pointed out: using back instead of rear for consistency.

File: packages/communication/ezrassor_keyboard_controller/nodes/keyboard_controller (hunk 2 above)
Comment (tigressine) [reply packages/communication/ezrassor_keyboard_controller/nodes/keyboard_controller]: Yeah I was leaning towards swapping out "back" to "rear" everywhere, but I don't actually remember why I preferred that. Doesn't really matter one way or the other, I'll change everything back to "back" :)

**Review by CSharpRon — APPROVED** [review]
Approve!

**Pull request merged**